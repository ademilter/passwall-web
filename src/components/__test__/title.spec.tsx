import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import PassTitle from '../title';

describe('<PassTitle /> with no props', () => {
  let container: ShallowWrapper;

  beforeEach(() => {
    container = shallow(<PassTitle />);
  });

  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

  it('should have logo', () => {
    expect(container.find('img[alt="Logo"]')).toBeTruthy();
  });
});
