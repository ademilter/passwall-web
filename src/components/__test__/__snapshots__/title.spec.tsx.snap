// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PassTitle /> with no props should match the snapshot 1`] = `"<h1 class=\\"ant-typography\\"><img src=\\"/images/icon_125.png\\" alt=\\"Logo\\" height=\\"40\\" width=\\"40\\" style=\\"margin:5px\\"/><span style=\\"font-weight:bold\\">Pass</span>Wall</h1>"`;
